tests/test_install.sh

```diff
@@ -17,50 +17,32 @@ rm -rf "$working_dir"
 mkdir -p "$working_dir"
 cd "$working_dir"
 
+# Start ephemeral network in the background
+network_live_time=60
+timeout --signal=SIGINT --kill-after=${network_live_time}s --preserve-status ${network_live_time}s \
+"$INSTALL_PREFIX"/bin/sandbox.sh --verbose &
+
+# Wait for service to be open
+sleep 45
+
+# # Issue tutorial transactions to ephemeral network
 python3.8 -m venv env
 source env/bin/activate
-python -m pip install -U -r "$INSTALL_PREFIX"/bin/requirements.txt
 python -m pip install ../../../python
+python ../../../python/tutorial.py ./workspace/sandbox_0/0.ledger/ ./workspace/sandbox_common/
 
 # Test Python package CLI
-../../test_python_cli.sh > test_python_cli.out
-
-# Start ephemeral network in the background
-network_info_file="network_info.txt"
-
-network_live_time=30
-timeout --signal=SIGINT --kill-after=${network_live_time}s --preserve-status ${network_live_time}s \
-python "$INSTALL_PREFIX"/bin/start_network.py \
-    -p liblogging \
-    -b "$INSTALL_PREFIX"/bin \
-    --library-dir ../../../build \
-    -g "$(pwd)"/../../../src/runtime_config/gov.lua \
-    --network-info-file "$network_info_file" \
-    -v &
-
-# Wait for network to be open and accessible
-while [ ! -f "$network_info_file" ]; do
-    sleep 1
-done
-
-# Issue tutorial transactions to ephemeral network
-python ../../../python/tutorial.py "$network_info_file"
+../../../tests//test_python_cli.sh > test_python_cli.out
 
 # Wait until original network has died
-sleep ${network_live_time}
-
-# ...and a tad longer to be sure
-sleep 5
+sleep 20
```

Comment:
Likewise, that seems like something the Python code in tutorial could more easily and precisely wait for.